opdr4.js

```diff
@@ -0,0 +1,54 @@
+//A
+
+function GetTvName(tv) {
```

Comment:
Functies in javascript dien je camelCase te schrijven
```suggestion
function getTvName(tv) {
```